src/java/org/learningequality/FullScreen.java

```diff
@@ -51,21 +42,6 @@ public void onPageFinished(WebView view, String url) {
                     loadingReady.run();
                     mInWelcome = true;
                 }
-
-                if (url.contains("en/explore/#/credits-section")) {
-                    Button button = new Button(mActivity);
-                    button.setText("Crash me");
-                    ViewGroup viewGroup = mActivity.getWindow().getDecorView().findViewById(android.R.id.content);
-                    button.setLayoutParams(new LinearLayout.LayoutParams(
-                            LinearLayout.LayoutParams.MATCH_PARENT,
-                            LinearLayout.LayoutParams.MATCH_PARENT));
-
-                    viewGroup.addView(button);
-
-                    button.setOnClickListener(v -> {
-                        throw new RuntimeException("This is a test crash");
-                    });
-                }
```

Comment:
Just to make things tidier, We should do a rebase so this change is in its own commit. Best thing might be to remove the change from this commit and instead add `git revert 150edf919d92b7cf953e895bdb645eaeb7bcd19e`.